app/models/conversation.rb

```diff
@@ -124,6 +124,10 @@ def can_reply?
     last_message_in_messaging_window?(messaging_window)
   end
 
+  def last_activity_at
```

Comment:
let's also add a spec to cover this case !!!

Reply:
Done! 8ee2419ea57ab62c936eb62a090f4fa392d1af1f